src/types/Bundle.ts

```diff
@@ -0,0 +1,4 @@
+export interface Bundle<T> {
+  total: number;
```

Comment:
> The `total` can be misinterpreted as the count of entries included in this response, i.e. identical to `entries.length`, but `total` actually includes the overall count of `T` on the server, apart from the arguments given in this request, right?

Correct! Can you clarify if this is a misinterpretation you experienced, or if you are worried about the possibility for misinterpretation? I ask because I would place much more weight on the former!

In general, I would hope we are building an API that does not needlessly repeat information, and hope that developers looking at this API for the first time would give us the grace of assuming we have that goal. Which is to say: we should document it ([as you mention below](https://github.com/PhilanthropyDataCommons/service/pull/296#discussion_r1164549895)), but I actually find `total` to be the clearest name for this concept, in part because it matches what FHIR is doing.

Reply:
Let me know if the description clears this up.

Reply:
Yes, the OpenAPI doc resolves it.